Dockerfile

```diff
@@ -39,7 +40,8 @@ CMD ["timestamp-server", "serve"]
 
 # debug compile options & debugger
 FROM deploy as debug
-RUN go install github.com/go-delve/delve/cmd/dlv@v1.9.0
+
+RUN go install -mod=mod github.com/go-delve/delve/cmd/dlv@v1.9.0
```

Comment:
I don't think this will work with hermetic builds, will it?

Reply:
Apologies I keep defaulting to getting things to build locally, these changes allowed me to build locally, I will revert them.